src/algo/astar.rs

```diff
@@ -25,8 +25,28 @@ use crate::visit::{EdgeRef, GraphBase, IntoEdges, Visitable};
 /// it should never overestimate the actual cost to get to the nearest goal node. Estimate costs
 /// must also be non-negative.
 ///
+/// Returns the total cost + the path of subsequent `NodeId` from start to finish, if one was
+/// found.
+///
 /// The graph should be `Visitable` and implement `IntoEdges`.
 ///
+///
+/// # Arguments
+/// * `graph`: weighted graph
+/// * `start`: the start node
+/// * `is_goal`: the callback defines the goal node
+/// * `edge_cost`: closure that returns cost of a particular edge
+/// * `estimate_cost`: closure that returns the estimated cost to the finish for particular node
```

Comment:
```suggestion
/// * `graph`: weighted graph.
/// * `start`: the start node.
/// * `is_goal`: the callback defines the goal node.
/// * `edge_cost`: closure that returns cost of a particular edge.
/// * `estimate_cost`: closure that returns the estimated cost to the finish for particular node.
```

Not sure about this, but that's how it is in your spfa example. Might apply to other algs aswell.